src/libmeasurement_kit/nettests/runnable.cpp

```diff
@@ -77,9 +77,15 @@ void Runnable::run_next_measurement(size_t thread_id, Callback<Error> cb,
     logger->debug("net_test: running with input %s", next_input.c_str());
     main(next_input, options, [=](Var<report::Entry> test_keys) {
         report::Entry entry;
+        // simple way to override the default entry["input"] behavior
+        if (!(*test_keys)["input"].is_null()) {
+            entry["input"] = (*test_keys)["input"];
+            test_keys->erase("input");
+        } else {
+            entry["input"] = next_input;
+        }
```

Comment:
Can you refactor the code such that the key inside of `*test_keys` is called `input_` rather than input to indicate that we're passing up information to be consumed at this layer? [This is something we already do in another case](https://github.com/measurement-kit/measurement-kit/blob/master/src/libmeasurement_kit/nettests/runnable.cpp#L164), so I'd rather see that for consistency.

Also, can you add `XXX` in front of the comment, to indicate that we can probably think at better solutions (but maybe not now, since the need just arose, and we'd better wait a little bit to figure out the best way, once more use cases emerge)?